EIPS/eip-requests.md

```diff
@@ -0,0 +1,74 @@
+---
```

Comment:
```suggestion
---
eip: 7685
```
Assigning next sequential EIP/ERC/RIP number.

Please also update the filename.